service/app/controllers/courseenrollment/CourseEnrollmentController.java

```diff
@@ -19,6 +19,7 @@
           request
               .getContext()
               .put(JsonKey.URL_QUERY_STRING, getQueryString(request().queryString()));
+          request.getContext().put(JsonKey.FIELDS, request().queryString().get(JsonKey.FIELDS));
```

Comment:
What if query string is not passed? Will request().queryString() be null?

Reply:
no it will be empty map ,